SPECS/mariner-rpm-macros/macros.python3

```diff
@@ -50,6 +50,11 @@
   rm -rfv %{buildroot}%{_bindir}/__pycache__
 }
 
+%py3_check() %{expand:\\\
```

Comment:
You'll now also need to update the signatures for this file and increment the `Release` in the spec itself.